frontend/src/components/Pages/RobotPage/AreaMapView.tsx

```diff
@@ -1,174 +0,0 @@
-import { CircularProgress, Typography } from '@equinor/eds-core-react'
-import { MouseEvent, useCallback, useEffect, useState } from 'react'
-import styled from 'styled-components'
-import NoMap from 'mediaAssets/NoMap.png'
-import { placeRobotInMap, inverseCalculatePixelPosition } from 'utils/MapMarkers'
```

Comment:
inverseCalculatePixelPosition is only used here. Implementation can also be deleted

Reply:
Done.